webhook_server_container/config/schema.yaml

```diff
@@ -51,6 +51,18 @@ properties:
       token:
         type: string
         format: password
+  strict:
```

Comment:
Should be under `branch-protection` section.

Reply:
Current code expects them to be similar to other global config option. Do you want that to change? How about repo ones? Should they be under branch protection section too?

Reply:
Yes, it should be under `branch-protection`.
What are `repo ones` that you mansion?

Reply:
ok, I moved them under branch_protection for both global and repo settings